sciences-po-ecole-doctorale-note-french.csl

```diff
@@ -393,7 +393,7 @@
               </else>
             </choose>
             <text term="cited" font-style="italic" suffix="."/>
-            <text variable="locator" prefix="p.&#160;"/>
+            <text variable="locator" prefix="p.&#160;" suffix="."/>
```

Comment:
This seems odd -- why do you want a period at the end only with a locator but not otherwise? This way, it'd also appear before the semicolon when multiple items are cited, is that right? It would be more common to set a `suffix="."` on `layout`

Reply:
Dear Sebastian,
Maybe this seems odd, but this is exactly how my institution want it..
Could you please validate this pull request ?
Best regards,
Anna